### DISCUSSION TITLE
Please tell me how to configure not to use EdDSA

### DISCUSSION THREAD

**Discussion by duolabmeng6** [sparkle-project/Sparkle#2214] [Q&A]
I don't want to use EdDSA on how to disable EdDSA before the application

I'm sorry to ask this question. I think EdDSA adds complexity to the release of updates. 

So I need to ban him. Is there an option like this? 

Do I modify the code to achieve

**Comment by zorgiepoo** [comment]
Using EdDSA signing on the update archives is strongly recommended for fallback and security. generate_appcast can be used to automatically make updates including generating delta updates (which requires EdDSA), which simplifies the process.

It's possible to not use EdDSA for regular app updates although this is not recommended, deprecated, and may be unsupported in a future release. You need to code sign your app with an Apple Developer ID certificate for this to work.
